File: testdata/MoveMethod/Cpp/parent/OrderProcessor.cpp
#include <vector>

class Item {
public:
    double getPrice() const {
        return price;
    }

private:
    double price;
};

class OrderProcessor {
public:
    void processOrder() {
        double total = calculateTotal();
        // Further order processing
    }

private:
    double calculateTotal() {
        double total = 0;
        for (const Item& item : items) {
            total += item.getPrice();
        }
        return total;
    }

    std::vector<Item> items;
};

File: testdata/MoveMethod/Java/parent/OrderProcessor.java
import java.util.List;

public class OrderProcessor {

    private List<Item> items;

    public void processOrder() {
        double total = calculateTotal();
        // Further order processing
    }

    public double calculateTotal() {
        double total = 0;
        for (Item item : items) {
            total += item.getPrice();
        }
        return total;
    }
}

class Item {
    private double price;

    public double getPrice() {
        return price;
    }
}
